src/lib/features/traffic-data-usage/traffic-data-usage-store-type.ts

```diff
@@ -7,6 +7,13 @@ export type IStatTrafficUsage = {
     count: number;
 };
 
+export type IStatMonthlyTrafficUsage = {
+    month: string;
```

Comment:
Yup, month makes more sense here then `day`, for example for `differenceInCalendarMonths(now, entry.day)` 👍🏻